project/src/main/editor/creature/creature-editor.gd

```diff
@@ -386,11 +388,6 @@ func _random_highlight_color(color: Color) -> Color:
 	return new_color
 
 
-func _on_Breadcrumb_trail_popped(_prev_path: String) -> void:
-	if not Breadcrumb.trail:
-		get_tree().change_scene("res://src/main/ui/menu/LoadingScreen.tscn")
-
-
 func _on_Quit_pressed() -> void:
```

Comment:
With this function removed, you can disconnect the signal to avoid an `Error calling method from signal 'trail_popped'` error

Reply:
Disconnected signal